pysrc/riffq/connection.py

```
@@ -0,0 +1,72 @@
+        else:
+            from pyarrow.cffi import export_stream
+            capsule = export_stream(reader)
+        callback(capsule)
+
+    def arrow_batch(self, values:list, names:list):
+        return pa.RecordBatchReader.from_batches(
+            pa.schema(list(zip(names, [v.type for v in values]))),
+            [pa.record_batch(values, names=names)],
+        )
+
+    def handle_auth(self, user, password, host, database=None, callback=callable):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)